ckb-bench/devtools/ci/ansible/playbook.yml

```diff
@@ -69,3 +69,25 @@
         - { src: "/var/lib/ckb-benchmark/data/ckb-bench.brief.md", dest: "ckb-bench.brief.md" }
       tags:
         - process_result
+
+
+    - name: Update and upgrade apt packages
+      become: true
+      become_method: sudo
+      apt:
+         upgrade: 'yes'
+         update_cache: yes
+      tags:
+         - apt_update
+
+    - name: Install prerequisites
+      become: true
+      become_method: sudo
+      apt:
+        name: ['libpq-dev', 'python3-psycopg2']
+        state: latest
+        install_recommends: yes
+        update_cache: yes
+      tags:
+        - apt_install_python_packages
+
```

Comment:
As we can access the test result from the hosted machine, we do not need to write the database from the remote machine via Ansible.

IMO, it should be done in `benchmark.sh`.

Reply:
can parsing the `ckb-bench.brief.md` directly   if do it in benchmark.sh, have already copy it into `GITHUB_WORKSPACE`.